redis/example/kustomization.yaml

```diff
@@ -2,14 +2,30 @@ apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 
 resources:
-  - ssh://github.com/utilitywarehouse/shared-kustomize-bases//redis/manifests/dev-enablement/redis-shared?ref=v0.0.0
+  - ssh://github.com/utilitywarehouse/shared-kustomize-bases//redis/manifests?ref=v0.0.0
+  #- ../manifests
 
 
-# Resource requests and limits
+# Add prefix to name
+namePrefix: service-cache-
+
+# OpsLevel annotations
+commonAnnotations:
+ "app.uw.systems/description": "cache for some service"
+ "app.uw.systems/tier": "tier_4"
+
 patches:
+  # replace namespace
+  - target:
+      namespace: change-me
+    patch: |-
+      - op: replace
+        path: /metadata/namespace
+        value: "some-namespace"
+  # Resource requests and limits
```

Comment:
Why do I want to change these? Have we found the defaults too high or too low? Asking because all the other changes in this example seem to have a clear reason, but not this one (or is this just an example of what you _can_ do?

Reply:
I added it in the example since changing the resources is most common use case. I hope it will save a few minutes to anyone using manifests